addons/product/report/product_product_templates.xml

```diff
@@ -123,9 +123,6 @@
                         <t t-else="">
                             <strong class="o_label_price_small" t-field="product.list_price" t-options="{'widget': 'monetary', 'label_price': True}"/>
                         </t>
-                        <div class="o_label_extra_data">
```

Comment:
to avoid xpath breaking, maybe we could leave it with eg. `t-if="0"` in stable (probability of an  issue seem very low so might be ok like this)

Reply:
Done. Then remove in master or leave as is?

Reply:
remove in master